### PR TITLE
Jakarta EE Platform 8 Specification

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.  (only use a single digit for platform spec)
- [x] Specification PDF in the form of wombat_x.y.pdf (only use a single digit for platform spec)
- [x] Specification HTML in the form of wombat_x.y.html (only use a single digit for platform spec)
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
https://oss.sonatype.org/content/repositories/staging/jakarta/platform/jakarta.jakartaee-api/8.0.0/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-jakartaeetck-8.0.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jakartaee-platform/issues/99

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.  (only use a single digit for platform spec)

### CODE REVIEW

File: platform/8.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Jakarta Platform Spec 8.0"
+date: 2019-07-18T16:10:38-04:00 (use "date +%FT%T%z")
```
Comment (bshannon) [inline platform/8.0/_index.md]: Don't forget to remove the "use" comment at the end for the final version.

File: platform/8.0/_index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+summary: "Initial release of the Jakarta EE 8 Platform"
+---
+The Jakarta EE Platform defines a standard platform for hosting Jakarta EE applications.
+
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.pdf) (PDF)
```
Comment (bshannon) [inline platform/8.0/_index.md]: We never discussed this explicitly but should we use "8.0" or just "8" for the version number?
For Java EE, we decided we would never do "dot" releases.

File: platform/8.0/_index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.html) (HTML)
+* Jakarta EE Platform 8.0 Javadoc - ??? (https://github.com/eclipse-ee4j/jakartaee-api)
```
Comment (bshannon) [inline platform/8.0/_index.md]: Just link to the apidocs directory that you'll include in a separate PR.

File: platform/8.0/_index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* Jakarta EE Platform 8.0 TCK - ???
+* Maven coordinates
+  * ???
```
Comment (bshannon) [inline platform/8.0/_index.md]: These should be the coordinates of the API jar file.

File: platform/8.0/_index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+|Payara                 |         |     |          |
+|Red Hat                |         |     |          |
+|Tomitribe              |         |     |          |
+|EE4J PMC               |         |     |          |
+|Participant Members    |         |     |          |
+|Committer Members      |         |     |          |
+
+# Compatible Implementations
+
+* Eclipse Glassfish 5.1
```
Comment (bshannon) [inline platform/8.0/_index.md]: Should probably link to an appropriate page.

File: platform/8.0/_index.md (hunk 1 above)
Comment (kwsutter) [reply platform/8.0/_index.md]: got it, thanks

File: platform/8.0/_index.md (hunk 2 above)
Comment (kwsutter) [reply platform/8.0/_index.md]: I kind of like the dot release, even for the Platform.  But, I'm flexible either way.  Maybe we should discuss at the Spec Committee?

File: platform/8.0/_index.md (hunk 3 above)
Comment (kwsutter) [reply platform/8.0/_index.md]: Sure.  Done.

File: platform/8.0/_index.md (hunk 4 above)
Comment (kwsutter) [reply platform/8.0/_index.md]: Left them as TBD for now.

File: platform/8.0/_index.md (hunk 5 above)
Comment (kwsutter) [reply platform/8.0/_index.md]: Good idea.  I wonder if the template should indicate that desire?

File: platform/8.0/_index.md (hunk 2 above)
Comment (bshannon) [reply platform/8.0/_index.md]: Yes, we should discuss, later...

**Comment by kwsutter** [comment]
# Spec Review Checklist

1. Spec PR
  - [ ] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [ ] Directory of form {spec}/x.y
  - [ ] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [ ] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
  - [ ] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [ ] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [ ] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [ ] Staging repository link of the form https://oss.sonatype.org/conte/repositories/staging/jakarta/{spec}/jakarta-{spec}-api/x.y.z/
  - [ ] EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [ ] Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [ ] Second PR for just apidocs

2. _index.md
  - [ ] Link to spec pdf
  - [ ] Link to spec html
  - [ ] Link to apidocs
  - [ ] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip
  - [ ] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [ ] Name of and link to at least one Compatible Implementation

3. javadocs
  - [ ] Footer contains Eclipse copyright and link to license
  - [ ] ESFL license is included, usually as doc-files/speclicense.html
  - [ ] no META-INF directory in PR
  - [ ] javadocs-jar artifact matches apidocs (optional for this release)

4. Spec PDF
  - [ ] Correct spec title
  - [ ] Version number of the form x.y, not x.y.z
  - [ ] Correct Eclipse copyright line
  - [ ] No DRAFT or SNAPSHOT
  - [ ] Correct Logo

5. Spec HTML
  - [ ] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [ ] EFTL license file, preferably named LICENSE.md
  - [ ] User's Guide (or equivalent documentation)
  - [ ] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [ ] Software requirements listed
  - [ ] Installation and configuration described
  - [ ] How to run tests
  - [ ] Where to file challenges

8. Compatibility certification request
  - [ ] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [ ] SHA-256 fingerprint matches staged TCK zip file
  - [ ] Request issue has `certification` label.

9. TCK results summary
  - [ ] Page is hosted by Compatible Implementation project
  - [ ] Includes all information from certification request
  - [ ] Summary includes number of tests passed, failed, errors
  - [ ] SHA-256 fingerprint matches staged TCK zip file on cert request

**Comment by kwsutter** [comment]
* On ballot completion
 - [x] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification and apidocs PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request.
 - [ ] The compatible implementation project/vendor MUST send an email tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as appropriate for the branch management for the project.

**Pull request merged**